app/schemas/students/response/timetable.py

```
@@ -0,0 +1,17 @@
+    course_name: str
+    course_identifier: str
+    professor: str
+    lecture_room: str
+    lecture_day: str
+    lecture_time: int
+    lecture_start_at: int
+    lecture_end_at: int
+
+
+class StudentTimetableResponse(BaseModel):
+    data: List[TimetableEntry]
```

Ruff (0.11.9)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)